app/helpers/reports_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ReportsHelper
-end
+    def today_to_check(data)
+        data ? "✅": "❌"
+    end
+end
```

Comment:
Add an empty space at the end of the file as a good practice

Reply:
Thanks